### PR TITLE
[WIP] Cassandra explicit

### PR DESCRIPTION
**Pull request by julialongtin**
in support of work on issue #327 , a cassandra-util cleanup.

### CODE REVIEW

File: libs/cassandra-util/src/Cassandra.hs [hunk 1]
```diff
@@ -1,4 +1,64 @@
+-- | The top import of our Cassandra utility library. Essentially, a "everyone who needs cassandra needs this" import.
 module Cassandra (module C) where
 
-import Cassandra.CQL  as C
-import Cassandra.Exec as C
+-- | pull in our extended wrapper of Database.CQL.Protocol
```
Comment (neongreen) [inline libs/cassandra-util/src/Cassandra.hs]: Not sure Haddock will parse that (`-- |` is for declarations and an `import` is not a declaration)

```suggestion
-- pull in our extended wrapper of Database.CQL.Protocol
```

File: services/brig/src/Brig/Provider/DB/Instances.hs [hunk 2]
```diff
@@ -2,12 +2,12 @@
 
 {-# OPTIONS_GHC -fno-warn-orphans #-}
 
-module Brig.Provider.DB.Instances () where
+module Brig.Provider.DB.Instances where
```
Comment (neongreen) [inline services/brig/src/Brig/Provider/DB/Instances.hs]:
```suggestion
module Brig.Provider.DB.Instances () where
```

Comment (neongreen) [reply services/brig/src/Brig/Provider/DB/Instances.hs]: The other one is `Galley.Data.Instances`

**Comment by neongreen** [comment]
(NB: I will be happy to approve it regarding consistency, etc., once there is consensus)

File: services/brig/src/Brig/Team/DB.hs [hunk 3]
```diff
@@ -26,7 +26,7 @@ import Brig.Types.Common
 import Brig.Types.User
 import Brig.Types.Team.Invitation
 import Galley.Types.Teams (Role)
-import Cassandra
+import Cassandra (MonadClient, BatchType(BatchLogged), Consistency(Quorum), PrepQuery, R, W, addPrepQuery, batch, emptyPage, hasMore, liftClient, paginate, paginateC, params, paramsP, query1, result, retry, setConsistency, setType, write, x1, x5)
```
Comment (ChrisPenner) [inline services/brig/src/Brig/Team/DB.hs]: This seems a smidge silly; I'm still in favour of being explicit in some way/shape/form, but qualified imports would probably be the way to go here. We really do need to decide on some standard for this tho 😋

**Comment by ChrisPenner** [comment]
Agreed with Artyom; until we have a concensus these sorts of things seem like a bit of a time-sink :/ 

Also these import lists are pretty ridiculously long; how long did it take to convert everything...?

I'm in favour of code consistency once we have an agreed-upon standard, but I don't know that it's the best use of time to go through the whole codebase and enforce them immediately; I'm in favour of incremental improvement 😄

**Pull request closed without merging**